decidim-surveys/app/models/decidim/surveys/survey_question.rb

```diff
@@ -3,7 +3,13 @@ module Decidim
   module Surveys
     # The data store for a SurveyQuestion in the Decidim::Surveys component.
     class SurveyQuestion < Surveys::ApplicationRecord
+      TYPES = %w(short_answer long_answer single_option multiple_option).freeze
+
       belongs_to :survey, class_name: Survey, foreign_key: "decidim_survey_id"
+
+      def answer_options
+        self[:answer_options] || []
```

Comment:
This smells like you're missing a `default: []` value in the migration that adds this column